cscs-checks/compile/libsci_resolve.py

```diff
@@ -76,6 +76,12 @@ def set_sanity(self):
                 sn.extractsingle(regex, self.stdout, 'version'), mod_name)
         ])
 
+    @rfm.run_before('compile')
+    def cray_linker_workaround(self):
```

Comment:
You haven't included the FIXME note in all the tests. Is this on purpose?

Reply:
Good point, I forgot it and I need to update the comment as well in the other fixme notes.

Reply:
@vkarak what do you think about adding this to XALT?

Reply:
@victorusu I don't understand this very well.